packages/react-art/src/__tests__/ReactART-test.js

```diff
@@ -15,22 +15,26 @@ const React = require('react');
 const ReactDOM = require('react-dom');
 const ReactTestUtils = require('react-dom/test-utils');
 
-let Group;
-let Shape;
-let Surface;
-let TestComponent;
-
-const Missing = {};
+// Isolate test renderer.
+jest.resetModules();
+const ReactTestRenderer = require('react-test-renderer');
 
+// Isolate ART renderer.
```

Comment:
It's unfortunate that forgetting to do this causes a totally non-obvious (to me) failure:
<img width="549" alt="screen shot 2018-05-18 at 2 24 12 pm" src="https://user-images.githubusercontent.com/29597/40258662-2e8d0580-5aa7-11e8-9e2a-1f08b434b18d.png">

I guess this is the only test where we need to do this though?

Reply:
Yeah. But good point. I think we might want to try to fail early although I’m not yet sure how.